modules/installation-vsphere-machines.adoc

```diff
@@ -14,13 +14,52 @@ To install {product-title} on user-provisioned infrastructure on VMware vSphere,
 
 .Prerequisites
 
-* Obtain the Ignition config files for your cluster.
-* Create a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
+* You have obtained the Ignition config files for your cluster.
+* You have access to an HTTP server that you can access from your computer and that the machines that you create can access.
+* You have created a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
 
 .Procedure
 
-. Convert the control plane, compute, and bootstrap Ignition config files to Base64 encoding.
-
+. Upload the bootstrap Ignition config file, which is named `<installation_directory>/bootstrap.ign`, that the installation program created to your HTTP server. Note the URL of this file.
++
+You must host the bootstrap Ignition config file because it is too large to fit in a vApp property.
```

Comment:
Since you have already removed "alternatively" step by using vApp, not sure if we still need to mention vApp here?

Reply:
Yes, that makes sense. This statement shouldn't be necessary without the alternative step of configuring vApp properties. Removed.